student-work/assignment-8/RIKI-CHOI/riki_script.py

```
@@ -0,0 +1,57 @@
+    print()
+    
+    # Random motivational message
+    motivations = [
+        "You've got this! =�",
+        "Keep pushing forward! =�", 
+        "Believe in yourself! P",
+        "Today is your day! <",
+        "Make it happen! ("
+    ]
+    
+    print(f"=� {random.choice(motivations)}")
```

Ruff (0.14.3)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)